### ISSUE TITLE
Feature: Allow Ignoring Parten Team

### ISSUE THREAD

**Issue by rowi1de** [equitybee/team-label-action#76]
Would be nice to ignore Parent Teams or Teams by label,
- e.g we have Department "Product" -> "Sub Team", Development -> "Sub Team" 
- We don't need "Product" or "Development" Labels

**Comment by drummerjolev** [comment]
Interesting idea! This is not a use-case we need to support internally currently so we probably won't work on this in the near future. Happy to review a PR if you want to take a stab at it.

**Comment by rowi1de** [comment]
ok thanks :)
Out of curiosity are all teams/squads at Top Level in your GitHub Org?

**Comment by drummerjolev** [comment]
Yes they are! At least for now.

**Comment by namoscato** [comment]
Yeah, this would be useful for for us as well and/or an allow list.

@drummerjolev, would you be open to both `allowed-labels` and `ignored-labels` input?

**Comment by drummerjolev** [comment]
Sure thing, I believe the `ignore-labels` would make sense. Why would you need both that one and `allow-labels` @namoscato?

**Comment by namoscato** [comment]
> Why would you need both that one and `allow-labels`

In our case, it'd be preferable to just define a (stricter) allow list, but we can work with `ignore-labels`.

Happy to implement this within the next ~week or so.

**Comment by drummerjolev** [comment]
Let's go with `ignore-labels` for now – looking forward to the PR!
